runtime/tests/jniarg/cdefs.c

```diff
@@ -139,8 +139,9 @@ void JNICALL Java_JniArgTests_logRetValError( JNIEnv *p_env, jobject p_this, jst
 	retValFailures ++;
 }
 
-void JNICALL Java_JniArgTests_summary( JNIEnv *p_env, jobject p_this )
+jint JNICALL Java_JniArgTests_summary( JNIEnv *p_env, jobject p_this )
 {
+   jint rc = 0;
```

Comment:
This appears misformatted - spaces instead of tab.

Reply:
Silly editors. Fixed in 5f938ef (see [force push](https://github.com/eclipse/openj9/compare/6d9d6bd876f3bef3b05bf6ab8eed33d3359c97e2..5f938ef88bb3f8d33239c8a3c05657370baa457e))